src/header/header.scss

```diff
@@ -119,9 +120,9 @@
 
   @media (max-width: 768px) {
     &,
-    &.iui-slim.iui-slim {
+    &.iui-slim {
       .iui-left {
-        .iui-header-button {
+        .iui-button {
```

Comment:
Why changed to target all buttons?

Reply:
I noticed it was not working for split header button (which does not use `iui-header-button` class)

Reply:
Maybe add that to test HTML.

Reply:
huh? it's already there

Reply:
Then why I don't see any difference? 😁

Reply:
this is inside a media query, it only happens on small screens